### PR TITLE
app-text/master-pdf-editor: version bump to 5.2.00

### PR DESCRIPTION
**Pull request by jpataias**
Version bump to 5.2.00

### CODE REVIEW

File: app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+COMMON_DEPEND="
+	>=media-gfx/sane-backends-1.0
+	>=dev-qt/qtsvg-5.4.1:5
```
Comment (a17r) [inline app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild]: Please keep deps sorted lexicographically.
Also, I'm quite sure you'll also need to depend on `qtcore` here.

Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild]: Given that qtsvg depends on qtcore is it really necessary? Just wondering.

Comment (a17r) [reply app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild]: yes, we don't rely on implicit deps.

Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild]: OK, just performed the required changes. Thanks.

File: app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+DESCRIPTION="A complete solution for viewing and editing PDF files"
+HOMEPAGE="https://code-industry.net/free-pdf-editor/"
+SRC_URI="https://code-industry.net/public/${P}_qt5.amd64.tar.gz"
+
+LICENSE="master-pdf-editor"
+SLOT="0"
+KEYWORDS="~amd64"
+RESTRICT="mirror"
+
+COMMON_DEPEND="
```
Comment (a17r) [inline app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild]: No `DEPEND`, but you use `COMMON_DEPEND` for some reason? If there is nothing built here, just make this whole block an `RDEPEND` instead.

Comment (jpataias) [reply app-text/master-pdf-editor/master-pdf-editor-5.2.00.ebuild]: Just included this change.

Happy New Year!

**Pull request closed without merging**